debian/control

```diff
@@ -30,8 +30,7 @@ Build-Depends: bison,
                liblz4-dev,
                liblzma-dev,
                liblzo2-dev,
-               libncurses5-dev (>= 5.0-6~),
-               libncurses5-dev:native (>= 5.0-6~),
+               libncurses-dev,
```

Comment:
This one should be handled in #2794 because it adds needed packages back in older distros